### ISSUE TITLE
Breaks on trailing comma (etc) in calls to method named `#[]`

### ISSUE THREAD

**Issue by bquorning** [samphippen/rubyfmt#94]
* Ruby version: `ruby 2.6.1p33 (2019-01-30 revision 66950) [x86_64-darwin18]`
* Rubyfmt git sha: de399cd09c50da975535ca27b974398576751421

## Input file

```ruby
Set[1,2,]
```

## Rubyfmt's output

```
Traceback (most recent call last):
	11: from src/rubyfmt.rb:2557:in `<main>'
	10: from src/rubyfmt.rb:2554:in `main'
	 9: from src/rubyfmt.rb:2397:in `format_program'
	 8: from src/rubyfmt.rb:2397:in `each'
	 7: from src/rubyfmt.rb:2398:in `block in format_program'
	 6: from src/rubyfmt.rb:2391:in `format_expression'
	 5: from src/rubyfmt.rb:2342:in `block in <main>'
	 4: from src/rubyfmt.rb:1830:in `format_aref'
	 3: from src/rubyfmt.rb:171:in `with_start_of_line'
	 2: from src/rubyfmt.rb:1834:in `block in format_aref'
	 1: from src/rubyfmt.rb:2391:in `format_expression'
src/rubyfmt.rb:2391:in `fetch': key not found: [:@int, "1", [1, 4]] (KeyError)
```

It breaks similarly on e.g. `Dir[File.expand_path "."]`

**Comment by bquorning** [comment]
Proposed fix in #95.

**Comment by samphippen** [comment]
Fixed via 315d17a
